### PR TITLE
Etiquette page updates

### PR DESCRIPTION
**Pull request by Schultea**
This PR implements the following **changes:**

* adding "other considerations" section as part of our FOIA update 

**URL / Link to page**

https://digital.gov/communities/manage-your-subscription/#general-etiquette

### CODE REVIEW

**Comment by Schultea** [comment]
Preview link: https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/AS_FOIA-Update/communities/manage-your-subscription/#general-etiquette

**Comment by Schultea** [comment]
@saracope would you like me to also change the page name to “Community Guidelines” vs "Manage Your Listserv Subscription"?

**Comment by Schultea** [comment]
Direct preview: https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/AS_FOIA-Update/communities/manage-your-subscription/#general-etiquette

File: content/communities/manage-listserv-subscription.md [hunk 1]
```diff
@@ -40,6 +40,18 @@ When sending a message to the entire Listserv, consider that your message is goi
 - Stay organized: Please keep the conversation threads focused. Start a new thread if it does not pertain to the latest conversation.
 - Be brief: Try to be direct and stay on point as much as possible.
 
+### Other considerations
```
Comment (afeijoo) [inline content/communities/manage-listserv-subscription.md:43]: I suggest making this an H2 instead of an H3.

**Review by afeijoo — DISMISSED** [review]

File: content/communities/manage-listserv-subscription.md (hunk 1 above)
Comment (Schultea) [reply content/communities/manage-listserv-subscription.md]: Just made an update!

**Comment by Schultea** [comment]
Only this missing from this PR is the name change to “Community Guidelines” and if you wanted to add “FOIA” to side navigation.

**Comment by saracope** [comment]
@Schultea Yes, please rename the page for now. The title only, not the URL. Also, move everything in 'Other Considerations' and put those items at the top of the 'Rules and Guidelines for Online Discussions' list. The 'Other Considerations' title isn't really needed. Then move all of the listserv info/content under the 'General Etiquette' sections.

**Comment by Schultea** [comment]
@saracope I made the updates [here](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/AS_FOIA-Update/communities/manage-your-subscription/#general-etiquette). Happy to move things around again if you like. I can also make "General Etiquette" into "Listserv Information" ... that may make more sense now.

**Comment by saracope** [comment]
@Schultea Yeah, I think you're right about changing 'general etiquette'. Looks great otherwise!

**Comment by Schultea** [comment]
Okay @saracope ready for you!

**Review by saracope — APPROVED** [review]

**Pull request merged**